setup.py

```diff
@@ -14,13 +14,13 @@
     author_email='opensource+pyramid-zipkin@yelp.com',
     license='Copyright Yelp 2016',
     url="https://github.com/Yelp/pyramid_zipkin",
-    description='Zipkin distributed tracing system support library for pyramid.',
+    description='Zipkin instrumentation for the Pyramid framework.',
     packages=find_packages(exclude=('tests*', 'testing*', 'tools*')),
```

Comment:
I know you didn't change this, but testing* and tools* don't exist; you can remove these.